.gitignore

```diff
@@ -1,5 +1,6 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 # dependencies
+package-lock.json
```

Comment:
we can't put package-lock json in git ignore

Reply:
Okay